### PR TITLE
Floor arm Subsystem and command list

### PR DESCRIPTION
**Pull request by jagernet-ops**

### CODE REVIEW

File: src/main/java/frc/robot/Constants.java [hunk 1]
```diff
@@ -71,6 +71,22 @@ public final class Constants {
       public static final float ARM_MAX_POSITION = 0;
       public static final float ARM_MIN_POSITION = 0;
 
+      //floor arm constants
+      public static final int FLOOR_ARM_SOLENOID_CHANNEL = 0;
+      public static final double FLOOR_ARM_POWER_SCALING = 0;
+      public static final int FLOOR_ARM_MOTOR = 13;
```
Comment (jdlanning) [inline src/main/java/frc/robot/Constants.java]: Needs a unique CAN id

File: src/main/java/frc/robot/subsystems/FloorArmSubsystem.java [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+import com.revrobotics.CANSparkMax.SoftLimitDirection;
+
+import edu.wpi.first.math.controller.PIDController;
+import edu.wpi.first.wpilibj.shuffleboard.Shuffleboard;
+import edu.wpi.first.wpilibj.shuffleboard.ShuffleboardTab;
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+import frc.robot.Constants;
+
+public class FloorArmSubsystem extends SubsystemBase {
+  private ShuffleboardTab floorArmTab = Shuffleboard.getTab("ArmSubsystem");
```
Comment (jdlanning) [inline src/main/java/frc/robot/subsystems/FloorArmSubsystem.java]: Rename tab to FloorArm

File: src/main/java/frc/robot/subsystems/FloorArmSubsystem.java [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+
+
+ 
+
+
+  
+
+  public FloorArmSubsystem() {
+    floorArmMotor = new CANSparkMax(Constants.FLOOR_ARM_MOTOR, MotorType.kBrushless);
+    pid = new PIDController(Constants.FLOOR_ARM_P_COEFF,Constants.FLOOR_ARM_I_COEFF,Constants.ARM_D_COEFF);
```
Comment (jdlanning) [inline src/main/java/frc/robot/subsystems/FloorArmSubsystem.java]: Make sure all constants in this file are renamed to FLOOR

File: src/main/java/frc/robot/commands/arm/MoveArm.java [hunk 4]
```diff
@@ -9,14 +9,14 @@
 
 import edu.wpi.first.wpilibj2.command.CommandBase;
 import frc.robot.Constants;
-import frc.robot.subsystems.ArmSubsystem;
+import frc.robot.subsystems.FloorArmSubsystem;
```
Comment (jdlanning) [inline src/main/java/frc/robot/commands/arm/MoveArm.java]: We don’t want this command to be FloorArm but still the original. It should be unchanged

File: src/main/java/frc/robot/commands/arm/MoveArmToPostion.java [hunk 5]
```diff
@@ -5,13 +5,13 @@
 package frc.robot.commands.arm;
 
 import edu.wpi.first.wpilibj2.command.CommandBase;
-import frc.robot.subsystems.ArmSubsystem;
+import frc.robot.subsystems.FloorArmSubsystem;
```
Comment (jdlanning) [inline src/main/java/frc/robot/commands/arm/MoveArmToPostion.java]: Same with this command , it should be unchanged

File: src/main/java/frc/robot/Constants.java (hunk 1 above)
Comment (jagernet-ops) [reply src/main/java/frc/robot/Constants.java]: Set the CAN id to 14 and changed the ID's for the FloorIntake and the ArmIntake to be 15 and 16 respectively.

File: src/main/java/frc/robot/subsystems/FloorArmSubsystem.java (hunk 2 above)
Comment (jagernet-ops) [reply src/main/java/frc/robot/subsystems/FloorArmSubsystem.java]: Renamed the Shuffleboard Title to FloorArm

**Review by jdlanning — APPROVED** [review]
LGTM

**Pull request merged**